src/server.tsx

```diff
@@ -98,10 +94,17 @@ const buildEpic = async (
         highlighted,
     };
 
-    // TODO - TARGETING
+    // Pass targeting data down to ContributionsEpicWrapper
+    // so it can determine whether to render the Epic or nothing
     const { html, css } = extractCritical(
         renderToStaticMarkup(
-            <ContributionsEpic content={content} tracking={tracking} localisation={localisation} />,
+            <ContributionsWrapper targeting={targeting}>
```

Comment:
Nice approach to use this wrapper component. I wonder if naming it even more explicitly would be worthwhile? `<ContributionsTargeting>` or something. What do you reckon?

Reply:
As discussed offline, I ended up getting rid of the wrapper since it's become redundant after all targeting logic was moved to the `lib/targeting.ts`. Don't think we're losing anything but I'm sure the wrapper pattern for this purpose will re-emerge at some point.